### PR TITLE
Docs on how to verify data availability on Ethereum network

### PR DESCRIPTION
**Pull request by 0xSasaPrsic**
Docs on how to verify data availability on the Ethereum network with code samples.

### CODE REVIEW

**Review by vthunder — APPROVED** [review]
Thanks! Left minor comments. Looks good to merge otherwise.

**Comment by vthunder** [comment]
Regarding my question about the dispatch — I'm not sure if it would be required on the testnet. Will we have a service calling dispatch periodically?

- If yes, we should tweak this doc to clarify that it's not needed (or clarify when it might be needed).
- If no, why not? Seems like an unnecessary step for the developer to have to call it.

File: docs/using-avail/use-cases-and-examples/validiums.md [hunk 1]
```diff
@@ -38,5 +41,544 @@ The Avail team will make this data availability verification simple on
...
+
+Function `submitData` receives `availApi` api instance, `data` that will be submitted,
+and the `account` which is sending the transaction. In order to create account
+it is necessary to create _keyring_ _pair_ for the account that wants to send the data.
+This can be done with `keyring.addFromUri(secret)` which creates keyring pair via suri
+(the secret can be a hex string, mnemonic phrase or a string).
+After creating keyring pair, it is possible to send data availability transaction
+`availApi.tx.dataAvailability.submitData(data);` to the Avail network. Once transaction is included in the block
+it is possible to dispatch data root by creating transaction
+`availApi.tx.daBridge.tryDispatchDataRoot(destinationDomain, bridgeRouterEthAddress, header);` with the parameters:
```
[the remark this answers is not in the document]
Comment (0xSasaPrsic) [reply docs/using-avail/use-cases-and-examples/validiums.md:86]: No, it is necessary to post tryDispatchDataRoot transaction which will post message to the home and act as inbox message which will get pick up by the off-chain agent. The other bridge implementation will not need this flow.

**Comment by 0xSasaPrsic** [comment]
> Regarding my question about the dispatch — I'm not sure if it would be required on the testnet. Will we have a service calling dispatch periodically?
> 
>     * If yes, we should tweak this doc to clarify that it's not needed (or clarify when it might be needed).
> 
>     * If no, why not? Seems like an unnecessary step for the developer to have to call it.

This is the design of the bridge and if something needs to be bridged, user must submit transaction. Not sure what was the initial discussion about the bridge architecture @prabal-banerjee

**Pull request merged**